### PR TITLE
chore: add publiccode.yml and set CI validations

### PR DESCRIPTION
**Pull request by bfabio**

### CODE REVIEW

File: publiccode.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+publiccodeYmlVersion: '0.2'
+categories:
+  - it-development
+description:
+  it:
+    features:
+      - Generazione di publiccode.yml
+      - Modifica di publiccode.yml
+      - Validazione di publiccode.yml
```
Comment (libremente) [inline publiccode.yml:13]:
```suggestion
      - Validazione di publiccode.yml
      - Importazione di publiccode.yml tramite URL
      - Possibilità di gestire diverse lingue
```

File: publiccode.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    genericName: Generatore di publiccode.yml
+    longDescription: |
+      This web app is meant to be used to generate a valid
```
Comment (libremente) [inline publiccode.yml:16]: Here we should have an Italian description

File: publiccode.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      Furthermore, the app can be used as a validator. In fact, it is possible
+      to paste or import an already existing `publiccode.yml` file inside the
+      editor. The editor will validate the imported document and help fix any
+      existing issue.
+    shortDescription: An editor to generate and validate a publiccode.yml file
+developmentStatus: stable
+it:
+  conforme:
+    gdpr: false
+    lineeGuidaDesign: false
```
Comment (libremente) [inline publiccode.yml:33]: Why false?

**Review by libremente — CHANGES_REQUESTED** [review]

File: publiccode.yml (hunk 3 above)
Comment (bfabio) [reply publiccode.yml:35]: Good point. I was not sure what that encompasses, is there an objective tool to evaluate conformance?

**Comment by bfabio** [comment]
Closing to try and trigger the GitHub action in #183

File: publiccode.yml (hunk 3 above)
Comment (bfabio) [reply publiccode.yml:32]: @libremente repopened as #183 to try and trigger the GitHub action to no avail. I'll file an issue for that.

**Pull request closed without merging**